arch/x86/crc_folding.c

```diff
@@ -16,7 +16,7 @@
  * For conditions of distribution and use, see copyright notice in zlib.h
  */
 
-#ifdef X86_PCLMULQDQ_CRC
+#if X86_PCLMULQDQ_CRC
```

Comment:
This is bad... Assumes X86_PCLMULQDQ_CRC is always defined

Reply:
I noticed that as soon as I made the PR. Thanks. It is resolved now.